core/src/scss/utilities/mixins/typography/_small-paragraph.scss

```diff
@@ -1,12 +1,11 @@
 @charset "UTF-8";
 
-//
-// @small-paragraph
-//
-// To be used for describing the content of an image.
-//
-// Style guide: Mixins.Typography.small-paragraph
-//
+///
+/// To be used for describing the content of an image.
```

Comment:
Is this what it is? We already have a caption typography style in Decanter. I thought it's just a generic font style that's slightly smaller than regular body font. We use it in Cards for the descriptions.

Reply:
@kerri-augenstein